gateways/node_gateway.py

```diff
@@ -60,7 +60,7 @@ def save_node(self, id: str, node: Node) -> Union[bool, None]:
         :return: If saved successfuly or not
         :rtype: bool
         """
-        return self.cache_client.set(NODE_COLLECTION_NAME, id, node.to_dict())
+        return self.cache_client.set(NODE_COLLECTION_NAME, id, node.to_dict(), NODE_CACHE_TTL)
```

Comment:
This won't fix the entries that are already there, right? Will we do a manual clean up?

Reply:
Yes, is in PR's TODO list